### ISSUE TITLE
List of azure functions to create

### ISSUE THREAD

**Issue by wobba** [CrayonAS/Cumulus#11]
* Add modern web part to page (PnP Templates does not support update scenario)
* Secondary owner (if only one owner in Group or in SP owner sec group)
* External users (enable/disable)
* Add Graph Metadata
* Delete O365 group, sp site
* Site logo (check if available using site designs for sites&groups)
* Rename site url + group mailbox (need to check if this is possible)
* Set group site title prefix + postfix
* Exclude from GAL (need to check if this is possible)
* Connect to Hub (?)
* Apply PnP Template

Create one task per function as needed - some functions already exist in Pzl/InMeta code base.

**Comment by olemp** [comment]
@stigre It's commited to https://github.com/CrayonAS/Cumulus.Monads now.

**Comment by wobba** [comment]
@olemp you can optimize to add owners/members in the same call as which you create the group. See https://stackoverflow.com/questions/46467617/create-a-group-in-microsoft-graph-api-with-a-owner

**Comment by Abdullahiibrahimahmed** [comment]
Done!  " Delete O365 group, sp site".

**Comment by olemp** [comment]
@Abdullahiibrahimahmed The related issue for **Improve CreateGroup with owners/members on first call** is https://github.com/CrayonAS/Cumulus.Monads/issues/1.

Can you do a pull request to the master branch?

**Comment by Abdullahiibrahimahmed** [comment]
@wobba, @olemp  is there any azure function in this list that i can assign my self which we dont have before.

**Comment by olemp** [comment]
Yes @Abdullahiibrahimahmed: https://github.com/CrayonAS/Cumulus.Monads/issues/16
